packages/interactive_media_ads/ios/interactive_media_ads/Package.swift

```diff
@@ -15,11 +15,15 @@ let package = Package(
   products: [
     .library(name: "interactive-media-ads", targets: ["interactive_media_ads"])
   ],
-  dependencies: [],
+  dependencies: [
+    .package(
+      url: "https://github.com/googleads/swift-package-manager-google-interactive-media-ads-ios",
+      from: "3.23.0")
```

Comment:
Do we want `upToNextMajor` instead?

Reply:
Does the iOS team already have policy on this? On Android, I think we only use pinned versions because gradle lacked the equivalent of a `Podfile.lock` or `pubspec.lock`. Should we be consistent with Android or should iOS implementations generally use `upToNextMajor` if possible?

cc @stuartmorgan

Reply:
Pinning is sort of a myth on Android because of optimistic upgrades. For CocoaPods we've generally allowed major version ranges for dependencies with an authoritative/trusted source (e.g., the vendor of the thing the plugin is wrapping).